src/components/screens/signMultiSignTransaction/summary/footer.js

```diff
@@ -28,22 +28,25 @@ export const ActionBar = ({
 );
 
 export const Feedback = ({
-  t, isMember, isFullySigned,
+  t, isMember, signatureStatus,
 }) => {
-  let feedback = 'Unknown error';
+  let feedback;
+  const statusMessages = {
+    fullySigned: t('Transaction is already fully signed.'),
+    occupiedByOptionals: t('Your signature will replace one optional signature.'),
+  };
   if (!isMember) {
     feedback = t('Only members of the group can sign the transaction.');
-  }
-  if (isFullySigned) {
-    feedback = t('Transaction is already fully signed.');
+  } else {
+    feedback = statusMessages[signatureStatus];
   }
 
   return (
     <BoxFooter
       direction="horizontal"
       className={styles.footer}
     >
-      <div className={`${styles.feedback} feedback`}>
+      <div className={`${styles.feedback} ${signatureStatus === 'occupiedByOptionals' ? styles.warning : styles.error} feedback`}>
```

Comment:
Prefer to use `signatureCollectionStatus.occupiedByOptionals`
```suggestion
      <div className={`${styles.feedback} ${signatureStatus === signatureCollectionStatus.occupiedByOptionals ? styles.warning : styles.error} feedback`}>
```